IDEAS/Examples/TwinHouses/BuildingO5_Exp1_1Port_PerfectGass.mo

```diff
@@ -0,0 +1,31 @@
+within IDEAS.Examples.TwinHouses;
+model BuildingO5_Exp1_1Port_PerfectGass
+  "Model for simulation of experiment 1 for the O5 building"
+ extends BuildingN2_Exp1(
+   bui=2,
+   exp=1,
+   redeclare BaseClasses.Structures.TwinhouseO5_AirPerfectGass
+                                                struct,
+   redeclare BaseClasses.Ventilation.Vent_TTH_pressureNetwork_AirPerfectGass vent,
+    sim(interZonalAirFlowType=IDEAS.BoundaryConditions.Types.InterZonalAirFlow.OnePort, n50=1.64,
+      A0=1,
+      a=0.15));
+
+  annotation (Icon(coordinateSystem(preserveAspectRatio=false)), Diagram(
+        coordinateSystem(preserveAspectRatio=false)),
+    experiment(
+      StartTime=15000000,
+      StopTime=23587200,
+      Interval=600,
+      Tolerance=1e-06,
+      __Dymola_Algorithm="Cvode"),
+    __Dymola_experimentSetupOutput(events=false),
+    __Dymola_experimentFlags(
+      Advanced(GenerateVariableDependencies=false, OutputModelicaCode=false),
+      Evaluate=true,
+      OutputCPUtime=true,
+      OutputFlatModelica=false),
+    __Dymola_Commands(file=
+          "modelica://IDEAS/Resources/Scripts/Dymola/Examples/TwinHouses/BuildingO5_Exp1.mos"
```

Comment:
The .mos file indicated in this line its the unit test for the model BuildingOO5_Exp1. Can you add a similar mos file for this model and adjust the path accordingly? Then this model is unit tested together with the others. I can add the reference results.